pyproject.toml

```diff
@@ -32,7 +32,8 @@ ujson = "^5"
 Jinja2 = { version = "^3", optional = true }
 numpy = [
     { version = "^1.24.2", optional = true, python = ">=3.9,<3.12" },
-    { version = "^1.26.2", optional = true, python = ">=3.12" },
+    { version = "^1.26.2", optional = true, python = "==3.12" },
```

Comment:
I'm not completely sure how this works. I.e. if `==3.12` would match any 3.12.x version or just 3.12.0. Do you know?